tfe.go

```diff
@@ -509,6 +510,38 @@ func (c *Client) newRequest(method, path string, v interface{}) (*retryablehttp.
 	return req, nil
 }
 
+// Encode encodes the values into ``URL encoded'' form
+// ("bar=baz&foo=quux") sorted by key.
+func EncodeIncludeParams(v url.Values) string {
```

Comment:
What do you think about adding unit tests for this function?

Reply:
Can do!